dts/riscv/it8xxx2.dtsi

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (c) 2020 ITE Corporation. All Rights Reserved.
+ * Copyright (c) 2019-2020 Jyunlin Chen <jyunlin.chen@ite.com.tw>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <dt-bindings/irq.h>
+
+/ {
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+		cpu@0 {
+			clock-frequency = <0>;
```

Comment:
the `clock-frequency` being 0 seems wrong.  If this should get set in the board dtsi than maybe remove the property here.

Reply:
OK, I'l remov it.  thanks.